mino/mod.go

```diff
@@ -48,19 +86,18 @@ type Players interface {
 	Len() int
 }
 
-// Sender is an interface to provide primitives to send messages to recipients.
-type Sender interface {
-	// Send sends the message to all the addresses. It returns a channel that
-	// will be populated with errors coming from the network layer if the
-	// message cannot be sent. The channel must be closed after the message has
-	// been/failed to be sent.
-	Send(msg serde.Message, addrs ...Address) <-chan error
-}
+// RPC is an abstraction of a distributed Remote Procedure Call.
+type RPC interface {
+	// Call is a basic request to one or multiple distant peers. It directly
+	// contacts all the players and thus expect a reasonable number of peers.
+	//
+	// The response channel must be closed after every request ended in a
+	// result, either a reply or an error.
```

Comment:
Am I understanding correctly? The phrasing is a bit ambiguous

```suggestion
	// The response channel must be closed once the request ends with a
	// result, either a reply or an error.
```